vpr/src/draw/intra_logic_block.cpp

```diff
@@ -237,11 +237,30 @@ static void draw_internal_load_coords(int type_descrip_index, t_pb_graph_node* p
             /* Find the number of instances for each child pb_type. */
             int num_pb = mode.pb_type_children[j].num_pb;
 
+            int num_blocks = num_pb * num_children;
+
+            // determine an optimal number of columns
+            int num_columns = 1;
+            for (int k = 1; k * k <= num_blocks; ++k) {
+                if (num_blocks % k == 0) {
+                    num_columns = k;
+                }
+            }
+            int num_rows = num_blocks / num_columns;
+
+            const int MAX_WIDTH_HEIGHT_RATIO = 2;
```

Comment:
Explain this.  Trying to make a squarish layout again?  Should this also consider num_rows and num_cols?